integration_test/third_party_apps_test.go

```diff
@@ -640,10 +640,14 @@ func fetchAppsAndMetadata(t *testing.T) map[string]metadata.IntegrationMetadata
 func modifiedFiles(t *testing.T) []string {
 	cmd := exec.Command("git", "diff", "--name-only", "origin/master")
 	out, err := cmd.Output()
+	stdout := string(out)
 	if err != nil {
-		t.Fatalf("got error calling `git diff`: %v", err)
+		stderr := ""
+        	if exitError := err.(*exec.ExitError); exitError != nil {
+			stderr = string(exitError.Stderr)
+		}
+		t.Fatalf("got error calling `git diff`: %v,stderr=%v,stdout=%v", err, stderr, stdout)
```

Comment:
Any reason we use commas here instead of newlines? Both `stdout` and `stderr` are likely to be multiline anyway.

Reply:
sure, changed to newlines.